### ISSUE TITLE
[Mage] Elemental Precision Inconsistency

### ISSUE THREAD

**Issue by FarmI3oy** [LightsHope/server#614]
![wowscrnshot_111317_103745_result](https://user-images.githubusercontent.com/33633204/32738116-925b8ce0-c861-11e7-9eb2-a629e1945bcb.jpg)
Since the ZG patch I've been experiencing a lot more resists in PvP and on level 60 targets as a frost mage. However, when equipping +hit gear the issue seems to go away. I believe that the Elemental Precision talent is not functioning as intended on level 60 targets. 

For example, the crocolisk pulls in ZG that every mage is farming these days is near impossible without hit gear equipped. I was averaging 4-5 resists per pull of 14 crocs without hit gear equipped. When I would equip hit gear over the cap amount my resists would drop to <1%. Coincidence?

What really is interesting is the resist rate in PvP. I'm going to post some screenshots of a recent example from just one encounter I had. I had multiple spell resists in a row + chilled being resisted indefinitely. 
![wowscrnshot_111317_103738_result](https://user-images.githubusercontent.com/33633204/32738115-90a1b6a4-c861-11e7-9adf-292d1454cd54.jpg)

**Comment by rezzidue** [comment]
Yes I complained believing it was My 2H staff from Arlokk in ZG that was broken. Devs said it was behaving correctly - when the problem was probably Elemental Precision talent.

I have had triple resists in PvP, which is unheard of.
I got 6 resists on the first pack in DM North. Six
I forgot the other examples I quoted to the devs but resists were insane.
I'm not sure what changed but it did feel like probably a third of my spells were getting resisted. Even now I am hit capped in pvp and get resists probably every 2nd player.
Possibly frost school related? Binary resists with the snare component?
